admin/sql/create_indexes.sql

```diff
@@ -2,10 +2,17 @@ BEGIN;
 
 CREATE INDEX mbid_ndx_lowlevel ON lowlevel (mbid);
 CREATE INDEX build_sha1_ndx_lowlevel ON lowlevel (build_sha1);
-CREATE UNIQUE INDEX data_sha256_ndx_lowlevel ON lowlevel (data_sha256);
+
+CREATE UNIQUE INDEX data_sha256_ndx_lowlevel_json ON lowlevel_json (data_sha256);
 
 CREATE INDEX mbid_ndx_highlevel ON highlevel (mbid);
-CREATE INDEX build_sha1_ndx_highlevel ON lowlevel (build_sha1);
+CREATE INDEX build_sha1_ndx_highlevel ON highlevel (build_sha1);
+
+CREATE UNIQUE INDEX data_sha256_ndx_highlevel_meta ON highlevel_meta (data_sha256);
+-- This index should not be unique on data, because it's possible to have the same estimate for
+-- two different recordings. Instead map it over data, model, and the id of the item
+CREATE INDEX data_sha256_ndx_highlevel_model ON highlevel_model (data_sha256, highlevel, model);
```

Comment:
Error here, should be `UNIQUE` again.
I think this is right. With a test migration I did I had two identical `data` fields for two different `hl.id`. A unique row in this table is a combination of the input file, the model used on it, and the data. If any of these change we need a new row.